sample/src/main/res/drawable/shape_rounded_grey.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <corners android:radius="@dimen/login_controls_corner_radius" />
+    <solid android:color="#F5F5F5" />
```

Comment:
Do we extract resources to dimens,xml colors.xml in the sample app? Or keep some of them in place for simplicity?

1. dimens (all | only dimens that occur in the layouts more than 1 time)
2. colors (all | only colors that occur in the layouts more than 1 time)

Reply:
My prefs:
Re 1. Only if they can be reused (more than 1 appearance)
Re 2. I'd prefer to keep all the colors in colors.xml because the UI design of sample app will be reusing probably all of them across different places of the app. The color palette seems simple and consistent across the screens.

Reply:
Extracted to grey_light. We need to define a proper color palette later